src/components/time-picker/time-picker.js

```diff
@@ -0,0 +1,97 @@
+import mixin from '../../globals/js/misc/mixin';
+import createComponent from '../../globals/js/mixins/create-component';
+import initComponentBySearch from '../../globals/js/mixins/init-component-by-search';
+
+class TimePicker extends mixin(createComponent, initComponentBySearch) {
+  /**
+   * Time picker UI.
+   * @extends CreateComponent
+   * @extends InitComponentBySearch
+   * @param {HTMLElement} element The element working as a time picker UI.
+   */
+  constructor(element, options) {
+    super(element, options);
+    this._initInputEvents();
+    this._setMinMaxValues();
+    this.element.militaryTime = false;
+  }
+
+  setMilitaryTime = () => {
+    this.element.militaryTime = true;
+    this._setMinMaxValues();
+  }
+
+  _setMinMaxValues = () => {
+    const hourInput = this.element.querySelector(this.options.classHoursInput);
+    if (this.element.militaryTime) {
+      hourInput.max = 24;
```

Comment:
@marijohannessen Some sources says that the “military time” wording is in use only in US/Canada. Also according to https://en.wikipedia.org/wiki/24-hour_clock#Military_time military time has more aspects than forcing 24-hour time. Now I realize you seem to have wanted to introduce "24-hour time” preference - If it’s the case I’d suggest two things:

1. Changing the wording
2. Put it in `this.options`, whose default is in the `options` static property